### PR TITLE
UserProfile cleaner fragments

### PR DESCRIPTION
**Pull request by jharm916**
.

### CODE REVIEW

File: app/src/main/java/com/cmsc436/quickbite/UserProfileActivity.java [hunk 1]
```diff
@@ -51,6 +51,8 @@ protected void onCreate(Bundle savedInstanceState) {
         // Gets username
         UserProfileActivity.username = ((MyApplication) this.getApplication()).getUsername();
 
+        UserProfileActivity.username = "Jeremy";
```
Comment (domthebigbomb) [inline app/src/main/java/com/cmsc436/quickbite/UserProfileActivity.java]: remove

File: app/src/main/res/layout/activity_userprofile_bites.xml [hunk 2]
```diff
@@ -2,67 +2,17 @@
...
-                android:layout_column="1"
-                android:text="Bites"
-                android:padding="3dip" />
-            <TextView
-                android:text="Tab"
-                android:gravity="right"
-                android:padding="3dip" />
-        </TableRow>
-    </TableLayout>
+    </android.support.v7.widget.CardView>
```
Comment (domthebigbomb) [inline app/src/main/res/layout/activity_userprofile_bites.xml:73]: You didnt put anything inside?

Comment (jharm916) [inline app/src/main/res/layout/activity_userprofile_bites.xml:73]: not done yet

Comment (domthebigbomb) [inline app/src/main/res/layout/activity_userprofile_bites.xml:73]: are you going to?

**Comment by domthebigbomb** [comment]
@jharm916 Update this PR asap

**Comment by domthebigbomb** [comment]
prayers

**Pull request merged**